### ISSUE TITLE
Can not run after upgrade from version 7.3.1 to 7.3.2

### ISSUE THREAD

**Issue by weiwxg** [abpframework/abp#17428]
```c#
 An unhandled exception has occurred while executing the request.
System.MissingMethodException: Method not found: 'System.Collections.Generic.IReadOnlyList`1<Volo.Abp.Settings.SettingDefinition> Volo.Abp.Settings.ISettingDefinitionManager.GetAll()'.
   at Volo.Abp.SettingManagement.SettingManagementStore.SetCacheItemsAsync(String providerName, String providerKey, String currentName, SettingCacheItem currentCacheItem)
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine)
   at Volo.Abp.SettingManagement.SettingManagementStore.SetCacheItemsAsync(String providerName, String providerKey, String currentName, SettingCacheItem currentCacheItem)
   at Volo.Abp.SettingManagement.SettingManagementStore.GetCacheItemAsync(String name, String providerName, String providerKey)
   at Volo.Abp.SettingManagement.SettingManagementStore.GetOrNullAsync(String name, String providerName, String providerKey)
   at Castle.DynamicProxy.AsyncInterceptorBase.ProceedAsynchronous[TResult](IInvocation invocation, IInvocationProceedInfo proceedInfo)
   at Volo.Abp.Castle.DynamicProxy.CastleAbpMethodInvocationAdapterWithReturnValue`1.ProceedAsync()
   at Volo.Abp.Uow.UnitOfWorkInterceptor.InterceptAsync(IAbpMethodInvocation invocation)
   at Volo.Abp.Castle.DynamicProxy.CastleAsyncAbpInterceptorAdapter`1.InterceptAsync[TResult](IInvocation invocation, IInvocationProceedInfo proceedInfo, Func`3 proceed)
   at Volo.Abp.Settings.TenantSettingValueProvider.GetOrNullAsync(SettingDefinition setting)
   at Volo.Abp.Settings.SettingProvider.GetOrNullValueFromProvidersAsync(IEnumerable`1 providers, SettingDefinition setting)
   at Volo.Abp.Settings.SettingProvider.GetOrNullAsync(String name)
   at Microsoft.AspNetCore.RequestLocalization.DefaultAbpRequestLocalizationOptionsProvider.GetLocalizationOptionsAsync()
   at Microsoft.AspNetCore.RequestLocalization.AbpRequestLocalizationMiddleware.InvokeAsync(HttpContext context, RequestDelegate next)
   at Microsoft.AspNetCore.Builder.UseMiddlewareExtensions.<>c__DisplayClass6_1.<<UseMiddlewareInterface>b__1>d.MoveNext()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)
```

**Comment by smartls** [comment]
I have same issue

**Comment by maliming** [comment]
Please upgrade all abp packages to 7.3.2. if you are using lepton please upgrade it as well.

**Comment by weiwxg** [comment]
> Please upgrade all abp packages to 7.3.2. if you are using lepton please upgrade it as well.

I think I have already done it, because my project upgrade use `abp update` command. But the problem remains.

>abp update
ABP CLI 7.3.2
Updating package "Volo.Abp.ObjectExtending" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.Account.Application" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.Account.Application.Contracts" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.AspNetCore.SignalR" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.BlobStoring" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.Identity.Application.Contracts" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.PermissionManagement.Application.Contracts" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.Identity.Application" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.Kafka" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.TenantManagement.Application.Contracts" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.PermissionManagement.Application" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.TenantManagement.Application" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.FeatureManagement.Application" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.FeatureManagement.Application.Contracts" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.SettingManagement.Application.Contracts" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.SettingManagement.Application" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.Autofac" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.Emailing" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.Identity.Domain" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.PermissionManagement.Domain.Identity" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.BackgroundJobs.Domain" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.AuditLogging.Domain" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.TenantManagement.Domain" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.FeatureManagement.Domain" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.SettingManagement.Domain" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.OpenIddict.Domain" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.PermissionManagement.Domain.OpenIddict" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.Identity.Domain.Shared" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.BackgroundJobs.Domain.Shared" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.AuditLogging.Domain.Shared" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.TenantManagement.Domain.Shared" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.FeatureManagement.Domain.Shared" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.PermissionManagement.Domain.Shared" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.SettingManagement.Domain.Shared" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.OpenIddict.Domain.Shared" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.Account.HttpApi" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.Identity.HttpApi" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.PermissionManagement.HttpApi" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.TenantManagement.HttpApi" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.FeatureManagement.HttpApi" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.SettingManagement.HttpApi" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.Account.HttpApi.Client" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.Identity.HttpApi.Client" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.PermissionManagement.HttpApi.Client" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.TenantManagement.HttpApi.Client" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.FeatureManagement.HttpApi.Client" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.SettingManagement.HttpApi.Client" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.PermissionManagement.MongoDB" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.SettingManagement.MongoDB" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.Identity.MongoDB" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.BackgroundJobs.MongoDB" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.AuditLogging.MongoDB" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.TenantManagement.MongoDB" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.FeatureManagement.MongoDB" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.OpenIddict.MongoDB" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.Http.Client.IdentityModel" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.Autofac" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.TestBase" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.Autofac" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.Authorization" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.AspNetCore.TestBase" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.AspNetCore.Mvc.UI.Theme.LeptonXLite" from v2.3.2 to v2.4.0-rc.1.
Updating package "Volo.Abp.BlobStoring.FileSystem" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.Autofac" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.Swashbuckle" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.AspNetCore.Serilog" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.Identity.Web" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.Account.Web.OpenIddict" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.TenantManagement.Web" from v7.3.1 to v7.3.2.
Updating package "Volo.Abp.SettingManagement.Web" from v7.3.1 to v7.3.2.
Volo packages are updated in Jee.Gc.HemmingForce solution.
Checking installed npm global packages...
Updated @abp/cropperjs to ~7.3.2 in \src\Jee.Gc.HemmingForce.Web\package.json.
Running Yarn on C:\Users\Administrator\Desktop\Code\Jee-cn\Gc-HemmingForce-App\src\Jee.Gc.HemmingForce.Web\
➤ YN0000: ┌ Resolution step
➤ YN0061: │ chokidar@npm:2.1.8 is deprecated: Chokidar 2 does not receive security updates since 2019. Upgrade to chokidar 3 with 15x fewer dependencies
➤ YN0061: │ fsevents@npm:1.2.13 is deprecated: The v1 package contains DANGEROUS / INSECURE binaries. Upgrade to safe fsevents v2
➤ YN0061: │ source-map-resolve@npm:0.5.3 is deprecated: See https://github.com/lydell/source-map-resolve#deprecated
➤ YN0061: │ source-map-url@npm:0.4.1 is deprecated: See https://github.com/lydell/source-map-url#deprecated
➤ YN0061: │ urix@npm:0.1.0 is deprecated: Please see https://github.com/lydell/urix#deprecated
➤ YN0061: │ resolve-url@npm:0.2.1 is deprecated: https://github.com/lydell/resolve-url#deprecated
➤ YN0032: │ nan@npm:2.17.0: Implicit dependencies on node-gyp are discouraged
➤ YN0002: │ @abp/bootstrap@npm:7.2.3 doesn't provide @popperjs/core (pbf509), requested by bootstrap
➤ YN0002: │ @abp/jquery-validation@npm:7.2.3 doesn't provide jquery (pcdee1), requested by jquery-validation
➤ YN0000: │ Some peer dependencies are incorrectly met; run yarn explain peer-requirements <hash> for details, where <hash> is the six-letter p-prefixed code
➤ YN0000: └ Completed in 7s 65ms
➤ YN0000: ┌ Fetch step
➤ YN0013: │ xtend@npm:4.0.2 can't be found in the cache and will be fetched from the remote registry
➤ YN0013: │ y18n@npm:3.2.2 can't be found in the cache and will be fetched from the remote registry
➤ YN0013: │ yallist@npm:4.0.0 can't be found in the cache and will be fetched from the remote registry
➤ YN0013: │ yargs-parser@npm:5.0.1 can't be found in the cache and will be fetched from the remote registry
➤ YN0013: │ yargs@npm:7.1.2 can't be found in the cache and will be fetched from the remote registry
➤ YN0000: └ Completed in 4s 718ms
➤ YN0000: ┌ Link step
➤ YN0007: │ @fortawesome/fontawesome-free@npm:5.15.4 must be built because it never has been before or the last one failed
➤ YN0007: │ es5-ext@npm:0.10.62 must be built because it never has been before or the last one failed
➤ YN0000: └ Completed in 13s 330ms
➤ YN0000: Done with warnings in 25s 232ms
Installing client-side packages...
Found 1 projects.
C:\Users\Administrator\Desktop\Code\Jee-cn\Gc-HemmingForce-App\src\Jee.Gc.HemmingForce.Web
Running Yarn on C:\Users\Administrator\Desktop\Code\Jee-cn\Gc-HemmingForce-App\src\Jee.Gc.HemmingForce.Web
➤ YN0000: ┌ Resolution step
➤ YN0002: │ @abp/bootstrap@npm:7.2.3 doesn't provide @popperjs/core (pbf509), requested by bootstrap
➤ YN0002: │ @abp/jquery-validation@npm:7.2.3 doesn't provide jquery (pcdee1), requested by jquery-validation
➤ YN0000: │ Some peer dependencies are incorrectly met; run yarn explain peer-requirements <hash> for details, where <hash> is the six-letter p-prefixed code
➤ YN0000: └ Completed in 0s 209ms
➤ YN0000: ┌ Fetch step
➤ YN0000: └ Completed in 0s 237ms
➤ YN0000: ┌ Link step
➤ YN0000: └ Completed in 0s 304ms
➤ YN0000: Done with warnings in 0s 857ms

**Comment by realLiangshiwei** [comment]
> Updating package "Volo.Abp.AspNetCore.Mvc.UI.Theme.LeptonXLite" from v2.3.2 to v2.4.0-rc.1.

You can manually change `2.4.0-rc.1` to `2.3.2`.

We fixed the problem,it will be available in 7.3.3: https://github.com/abpframework/abp/pull/17344
